tests/test_board.py

```diff
@@ -57,32 +58,25 @@ def test_returns_all_spaces_when_none_have_been_occupied(board):
     assert len(board.available_spaces()) == number_of_spaces
 
 
-def test_only_returns_unoccupied_spaces_when_some_have_been_occupied(board):
-    board.place_marker(0, 'X')
+def test_does_not_return_occupied_spaces(board):
```

Comment:
Not sure how this test is any different from `test_a_space_is_unavailable_when_it_exists_but_is_occupied`?